gateway-package/src/main/resources/bin/start.sh

```diff
@@ -97,6 +97,11 @@ LIBPATH="$LIBPATH":"${JAVA_HOME}"/lib/s390/default
 LIBPATH="$LIBPATH":"${JAVA_HOME}"/lib/s390/j9vm
 LIBPATH="$LIBPATH":"${LIBRARY_PATH}"
 
+if [[ ! -z ${ZWE_GATEWAY_LIBRARY_PATH} ]]
+then
+    LIBPATH="$LIBPATH":"${ZWE_GATEWAY_LIBRARY_PATH}"
```

Comment:
It is already possible to specify the library path using https://github.com/zowe/api-layer/blob/e9316f5744ca361457aee5b4954719b5f55c8480/gateway-package/src/main/resources/bin/start.sh#L43 , it would be better to add libraries to the directory already used for this purpose or modify this value

Reply:
@achmelo `LIBRARY_PATH` is used by every start script, so setting an extension for one service would set it for all - is this worth avoiding 2 library path variables?